test-packages/e2e-tests/test/batch.spec.ts

```diff
@@ -15,6 +17,38 @@ describe('batch', () => {
   beforeEach(async () => deleteEntity(entityKey, destination));
   afterEach(async () => deleteEntity(entityKey, destination));
 
+  it('should execute multiple function imports', async () => {
+    await deleteEntity(1234, destination);
```

Comment:
[q] Not sure I understand why we call `deleteEntity` here.

Reply:
[req] I guess this is not intended.